ftw/copymovepatches/tests/test_catalog_fixes_view.py

```diff
@@ -32,6 +34,7 @@ def test_rebuild_broken_uid_index(self, browser):
             browser.css('.uid-index-state-check').first.text)
 
     @browsing
+    @skipIf(IS_PLONE_5, 'We no longer create AT content within DX.')
```

Comment:
@jone, is this assumption correct? If yes, does it mean that the brain metadata won't be broken at all?

Reply:
It is true that the specific use case described in the readme (AT in DX) does not happen when we have no AT anymore.
I'm not sure though if there are any use cases which may cause a similar problem.
If possible, I'd try to keep the test with DX.

What happens when running the test with DX only? Is there a problem with changing the UID without indexing?

Reply:
The problem was the code which changed the uid of the object in the test: it only worked for AT. I have added a DX version, see [my other comment](https://github.com/4teamwork/ftw.copymovepatches/pull/11/files#r160102408).